examples/image.rs

```diff
@@ -54,20 +61,25 @@ fn main() {
 }
 
 fn run(
-    dkr_ref: &reference::Reference,
+    registry: &str,
+    image: &str,
+    version: &str,
     user: Option<String>,
     passwd: Option<String>,
 ) -> Result<(), boxed::Box<error::Error>> {
+    env_logger::Builder::new()
```

Comment:
I'm surprised this works (here and in other examples as well) as I don't see imports for either `env_logger` nor `log`. Do you know what's the magic trick here?

Reply:
I can't tell you. I went ahead and removed all `extern crate log/env_logger` from the examples and it still works.

Reply:
It looks like this is an ergonomic improvement from 1.30 module rework: https://blog.rust-lang.org/2018/10/25/Rust-1.30.0.html#module-system-improvements. It works here because we are using the full (namespaced) path.